cms/envs/common.py

```diff
@@ -107,6 +107,9 @@
     # Enterprise service settings
     ENTERPRISE_CATALOG_INTERNAL_ROOT_URL,
 
+    # Disucssions settings
+    ENABLED_DISCUSSION_PROVIDERS,
```

Comment:
Unlike other plugins, we have a explicit list of enabled providers here, so that unused providers can be disabled. This is only useful if a lot of providers ship with the platform itself. If not, this isn't really needed, since people can just install/uninstall providers.